frontend/packages/console-shared/src/components/toast/ToastProvider.scss

```diff
@@ -0,0 +1,5 @@
+.ocs-toast-provider {
+  &.pf-m-toast {
+    list-style: none;
```

Comment:
I believe @sg00dwin has a general fix for list styles in console. We should go ahead with this fix for now, but we might be able to remove this style after @sg00dwin's fix lands.

Reply:
yeah thanks and i'll check with update once we have generic fix